util/dvsim/Testplan.py

```diff
@@ -678,9 +715,10 @@ def get_test_results_table(self, map_full_testplan=True):
                 if tr.total == 0 and not map_full_testplan:
                     continue
                 pass_rate = self._get_percentage(tr.passing, tr.total)
+
                 table.append([
-                    milestone, tp_name, tr.name, tr.passing, tr.total,
-                    pass_rate
+                    milestone, tp_name, tr.name, tr.passing, tr.total, pass_rate,
+                    tr.job_runtime.__str__(), tr.simulated_time.__str__()
```

Comment:
```suggestion
                    str(tr.job_runtime), str(tr.simulated_time)
```

Reply:
Oh! That is why we are modifying this __str__ function! Thanks!